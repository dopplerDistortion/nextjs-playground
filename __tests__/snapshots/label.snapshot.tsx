import {Label} from '@radix-ui/react-label';
import {queryByLabelText, render} from '@testing-library/react';

const DummyLabel = () => <Label>Your email address</Label>;
it('renders label unchanged', () => {
  const {container} = render(<DummyLabel />);
  expect(container).toMatchSnapshot();
});
