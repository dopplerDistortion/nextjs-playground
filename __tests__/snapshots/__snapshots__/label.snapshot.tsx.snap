// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders label unchanged 1`] = `
<div>
  <label>
    Your email address
  </label>
</div>
`;
